ReadMe Updated

jura02 opened the pull request.

jura02 committed:
Create ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

jura02 committed:
Update ReadMe.md

Rauldg committed:
Correct wording of some sentences and markdown

Rauldg commented on ReadMe.md:
```diff
@@ -0,0 +1,86 @@
+
+Answer "whether C++11 should be enabled for Rock packages [false]" with "true" and the other with default (just hit enter)
+    
+Sometimes you may face the problems like "No Access Rights to your repository": Make sure you have corrected generated the SSH Key for both GitLab and GitHub
```
Please Move all this to the Troubleshooting section:

> Sometimes you may face the problems like "No Access Rights to your repository": Make sure you have corrected generated the SSH Key for both GitLab and GitHub

> Similarly, sometimes you don't have access to the specific Repository forexample:
"No Access Rights to git@git.hb.dfki.de/models-environments" 
In this case, you have to make an access request to the maintainer/owner of that Repository.

jura02 replied:
Done.

jura02 committed:
Update ReadMe.md

Rauldg committed:
Improve sentences and instructions

Proof read and instruction for the creation of the folder to allocate the environment
